Practical 5 - Dictionaries, Code Review/state_names.py

```diff
@@ -0,0 +1,53 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
```

Comment:
Probably left over from original prac

Reply:
Hi @cforrest96 - Thanks fro reviewing Jayden's work.
Please see my comments below, and perhaps on your next code review you can look for things like good naming as well.